Could not find artifact org.infinispan:infinispan-bom:pom:9.0.0-SNAPSHOT

If people do not setup `<repositories/>` in `pom.xml` then running OOB Maven build yields an error:

````
$ git clone https://github.com/slaskawi/infinispan-wf-swarm-example.git
$ cd infinispan-wf-swarm-example/
$ mvn clean package
[INFO] Scanning for projects...
Downloading: https://repo.maven.apache.org/maven2/org/wildfly/swarm/bom-all/2016.9/bom-all-2016.9.pom
Downloaded: https://repo.maven.apache.org/maven2/org/wildfly/swarm/bom-all/2016.9/bom-all-2016.9.pom (15 KB at 2.7 KB/sec)
[ERROR] [ERROR] Some problems were encountered while processing the POMs:
[ERROR] Non-resolvable import POM: Could not find artifact org.infinispan:infinispan-bom:pom:9.0.0-SNAPSHOT @ line 28, column 25
[ERROR] 'dependencies.dependency.version' for org.infinispan:infinispan-embedded:jar is missing. @ line 63, column 21
[ERROR] 'dependencies.dependency.version' for javax.cache:cache-api:jar is missing. @ line 67, column 21
 @ 
[ERROR] The build could not read 1 project -> [Help 1]
[ERROR]   
[ERROR]   The project org.infinispan:wildfly-swarm-demo:1.0.0-SNAPSHOT (/Users/lvlcek/projects/infinispan-wf-swarm-example/pom.xml) has 3 errors
[ERROR]     Non-resolvable import POM: Could not find artifact org.infinispan:infinispan-bom:pom:9.0.0-SNAPSHOT @ line 28, column 25 -> [Help 2]
[ERROR]     'dependencies.dependency.version' for org.infinispan:infinispan-embedded:jar is missing. @ line 63, column 21
[ERROR]     'dependencies.dependency.version' for javax.cache:cache-api:jar is missing. @ line 67, column 21
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/ProjectBuildingException
[ERROR] [Help 2] http://cwiki.apache.org/confluence/display/MAVEN/UnresolvableModelException
````

After I [configured JBoss repository](https://developer.jboss.org/wiki/MavenGettingStarted-Developers) in `pom.xml` then Maven was able to download missing dependencies and package the project correctly.

I.e. add the following to `pom.xml`

````xml
<repositories>  
    <repository>  
      <id>jboss-public-repository-group</id>  
      <name>JBoss Public Repository Group</name>  
      <url>http://repository.jboss.org/nexus/content/groups/public/</url>  
      <layout>default</layout>  
      <releases>  
        <enabled>true</enabled>  
        <updatePolicy>never</updatePolicy>  
      </releases>  
      <snapshots>  
        <enabled>true</enabled>  
        <updatePolicy>never</updatePolicy>  
      </snapshots>  
    </repository>  
  </repositories>  
  <pluginRepositories>  
    <pluginRepository>  
      <id>jboss-public-repository-group</id>  
      <name>JBoss Public Repository Group</name>  
      <url>http://repository.jboss.org/nexus/content/groups/public/</url>  
      <releases>  
        <enabled>true</enabled>  
      </releases>  
      <snapshots>  
        <enabled>true</enabled>  
      </snapshots>  
    </pluginRepository>  
  </pluginRepositories>
````

--- a/pom.xml
+++ b/pom.xml
@@ -81,4 +81,35 @@
             <artifactId>infinispan</artifactId>
         </dependency>
     </dependencies>
+
+    <repositories>
+        <repository>
+            <id>jboss-public-repository-group</id>
+            <name>JBoss Public Repository Group</name>
+            <url>http://repository.jboss.org/nexus/content/groups/public/</url>
+            <layout>default</layout>
+            <releases>
+                <enabled>true</enabled>
+                <updatePolicy>never</updatePolicy>
+            </releases>
+            <snapshots>
+                <enabled>true</enabled>
+                <updatePolicy>never</updatePolicy>
+            </snapshots>
+        </repository>
+    </repositories>
+    <pluginRepositories>
+        <pluginRepository>
+            <id>jboss-public-repository-group</id>
+            <name>JBoss Public Repository Group</name>
+            <url>http://repository.jboss.org/nexus/content/groups/public/</url>
+            <releases>
+                <enabled>true</enabled>
+            </releases>
+            <snapshots>
+                <enabled>true</enabled>
+            </snapshots>
+        </pluginRepository>
+    </pluginRepositories>
+    
 </project>

